lib/Service/ExAppConfigService.php

```diff
@@ -43,20 +54,28 @@ public function getAppConfigValues(string $appId, array $configKeys): ?array {
 
 	public function setAppConfigValue(string $appId, string $configKey, mixed $configValue, ?int $sensitive = null): ?ExAppConfig {
 		$appConfigEx = $this->getAppConfig($appId, $configKey);
+		if ($sensitive) {
+			try {
+				$encryptedValue = $this->crypto->encrypt($configValue);
```

Comment:
empty string check can be done here in the `if` otherwise empty string values cannot be set for sensitive fields. It would throw an exception and return.